components/x-follow-button/src/FollowButton.jsx

```diff
@@ -51,6 +52,10 @@ export const FollowButton = (props) => {
 					token: csrfToken
 				};
 
+				if (typeof onClick === 'function') {
+					onClick(detail);
```

Comment:
I do think it's a little odd that this is an `onClick` prop but it's called `onSubmit`. It would be more logical to either move it onto the actual button, or keep it where it is but rename it to `onSubmit`?

Having said that, in this case it probably doesn't make much difference practically.

Reply:
updated